features/old/services/subscriptions/bulk_operations/send_emails.feature

```diff
@@ -25,34 +25,29 @@ Feature: Mass email bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
     Given I am logged in as provider "foo.3scale.localhost"
 
-  Scenario: Emails can be sent without body
+  Scenario: Emails can't be sent without body
     Given provider "foo.3scale.localhost" has "service_plans" visible
     And I am on the service contracts admin page
     And a clear email queue
     When I check select for "jane"
     And I press "Send email"
     And I fill in "Subject" with "Nothing to say"
     And I fill in "Body" with ""
-    And I press "Send" and I confirm dialog box within colorbox
-    Then I should see "Action completed successfully"
-    And "jane@me.us" should receive an email with the following body:
-    """
-    """
+    And I press "Send"
+    Then I should see "Selected Service Subscriptions"
+      And a clear email queue
```

Comment:
This is clearing the email queue rather than asserting it's empty, see: https://github.com/3scale/porta/blob/f6e98d1d8a66f28d13a91ebba8767e12814509ae/features/step_definitions/email_steps.rb#L49-L51

Reply:
uisng **no emails have been sent** will be correct, because this is what I am checking.

Reply:
@josemigallas Updated